acceptance/bundle/libraries/pypi/out.job.create.txt

```diff
@@ -0,0 +1,52 @@
+{
+  "method": "POST",
+  "path": "/api/2.1/jobs/create",
+  "body": {
+    "deployment": {
+      "kind": "BUNDLE",
+      "metadata_file_path": "/Workspace/Users/[USERNAME]/.bundle/pypi/default/state/metadata.json"
+    },
+    "edit_mode": "UI_LOCKED",
+    "format": "MULTI_TASK",
+    "max_concurrent_runs": 1,
+    "name": "test-job",
+    "queue": {
+      "enabled": true
+    },
+    "tasks": [
+      {
+        "dbt_task": {
+          "commands": [
+            "dbt deps --target=default",
+            "dbt seed --target=default --vars \"{ dev_schema: [USERNAME] }\"",
+            "dbt run --target=default --vars \"{ dev_schema: [USERNAME] }\""
+          ],
+          "profiles_directory": "dbt_profiles/",
+          "project_directory": "/Workspace/Users/[USERNAME]/.bundle/pypi/default/files",
+          "schema": "default"
+        },
+        "libraries": [
+          {
+            "pypi": {
+              "package": "dbt-databricks>=1.8.0,<2.0.0"
```

Comment:
Is this what you expect to see here? In that case, can you either add a comment or perhaps jq filter to only show this section.